src/exercise/05.js

```diff
@@ -4,24 +4,32 @@
 import * as React from 'react'
 import '../box-styles.css'
 
-// 💰 Use the className for the size and style (backgroundColor) for the color
-// 💰 each of the elements should also have the "box" className applied
+const Box = ({size, style, children}) => {
+  const classes = `box box--${size}`;
+  const styles = {
+    ...style,
+    fontStyle: 'italic'
+  };
 
-// 🐨 add a className prop to each of these and apply the correct class names
-// 💰 Here are the available class names: box, box--large, box--medium, box--small
-
-// 🐨 add a style prop to each of them as well so their background color
-// matches what the text says it should be as well as `fontStyle: 'italic'`
-const smallBox = <div>small lightblue box</div>
-const mediumBox = <div>medium pink box</div>
-const largeBox = <div>large orange box</div>
+  return (
+    <div className={classes} style={styles}>
+      {children}
+    </div>
+  );
+};
 
 function App() {
   return (
     <div>
-      {smallBox}
-      {mediumBox}
-      {largeBox}
+      <Box size="small" style={{backgroundColor: 'lightblue'}}>
+        small lightblue box
+      </Box>
+      <Box size="medium" style={{backgroundColor: 'pink'}}>
+        medium pink box
+      </Box>
+      <Box size="large" style={{backgroundColor: 'orange'}}>
+        large orange box
+      </Box>
```

Comment:
The directions don't really say to do this, but if you want you could try adding a "sizeless" box here to see what happens: `<Box>sizeless box</Box>` (the answer key includes this if you want an example!)

Reply:
That's great! I really like `PropTypes` – we use them heavily in the NYP Membership plugin! :) I'm going through the final versions now, looking at differences between them and my solutions. It's very helpful! Thanks for the support Kaitlin! 💖